apps/feedback/lib/mailer.ex

```diff
@@ -109,8 +109,10 @@ defmodule Feedback.Mailer do
     end
   end
 
-  defp formatted_utc_timestamp(date) do
+  @spec formatted_utc_timestamp(map) :: String.t()
+  def formatted_utc_timestamp(date) do
```

Comment:
This should be `DateTime.t()`, since the type of `Feedback.Message.incident_date_time` is also `DateTime.t()`.
You could then also use `DateTime.shift_zone!("Etc/UTC")` instead of going through the Timex overhead.